internal/cli/root/atlas/builder.go

```diff
@@ -88,7 +88,8 @@ func Builder(profile *string) *cobra.Command {
 			"toc": "true",
 		},
 		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
-			if config.Service() == "" {
+			if shouldSetService(cmd) {
+				_, _ = fmt.Printf("service not set. Atlas CLI used '%s'\n\n", config.CloudService)
```

Comment:
this is a breaking change, tools that parse the output are going to break

Reply:
I don't see how to prevent this if we want to print a warning message 😕

Reply:
setting the default to cloud for atlas commands it's been there even before `atlas cli` no one has been surprised, I don't believe this warning adds any value but if product wants it then use sdterr instead of stdout

Reply:
@boooczek We need your input to decide if we want to print a message to stderr if the user did not provide the service as we're automatically setting the service = cloud. As Gustavo said, we have been always automatically setting service = cloud when service is not set without printing any message. If you agree to print this message, I also need your input to decide what we should show to the user. The current message is 

> service not set. MongoCLI used 'cloud'

but `cloud` may be confusing.

Reply:
Discussed with the product manager and they decided to not show the message as we have been always doing. @colm-quinn

Reply:
sgtm, thanks for the tag!